Game/ConsoleApplication2/Game.cpp

```diff
@@ -3,6 +3,25 @@
 
 Game::Game()
 {
+	GameObject* player = new Player(Rect(Vector2D(950, 530), Vector2D(20, 20)),	//bounds
+									Vector2D(800, 800),							//accel
+									Vector2D(400, 400),							//maxSpeed
+									true);										//isMiniMapObject
+ 
+	InputManager* input = InputManager::getInstance();
+
+	input->AddListener(static_cast<int>(EventListener::KeyDownEvent::UP), player);
+	input->AddListener(static_cast<int>(EventListener::KeyDownEvent::LEFT), player);
+	input->AddListener(static_cast<int>(EventListener::KeyDownEvent::DOWN), player);
+	input->AddListener(static_cast<int>(EventListener::KeyDownEvent::RIGHT), player);
+	input->AddListener(static_cast<int>(EventListener::KeyUpEvent::UP), player);
```

Comment:
AddListener() takes an int, and the event types are part of an enum class so they need to be casted.

Reason I used enum classes was so that I would be able to distinguish between the different event types for the onEvent() method.